util/dvsim/Deploy.py

```diff
@@ -65,6 +65,11 @@ def __init__(self, sim_cfg):
         # A list of jobs on which this job depends
         self.dependencies = []
 
+        # Flat to indicates whether running is job requires all dependencies
```

Comment:
```suggestion
        # Indicates whether running this job requires all dependencies to pass.
```

Reply:
Oops. I typed `Flat` instead of `Flag`.. Thanks for the suggestion!